packages/donate-button-beta/src/index.tsx

```diff
@@ -0,0 +1,40 @@
+import {render} from 'preact';
```

Comment:
I just moved the logic for making the donate button into this file since it's so short.